### PR TITLE
Starting to write DAO tests.

### PR DESCRIPTION
**Pull request by kmp3325**
@tuttleb Figured out a way to get DAO tests going.  Pretty simple solution.

### CODE REVIEW

File: src/test/java/accounting/data/TransactionDAOTest.java [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+import org.skife.jdbi.v2.DBI;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
 public class TransactionDAOTest {
+    private TransactionDAO dao;
+
+    @Before
+    public void setup() {
+        DBI dbi = new DBI("jdbc:mysql://localhost:3306/accounting", "root", "root");
```
Comment (tuttleb) [inline src/test/java/accounting/data/TransactionDAOTest.java]: Does this mean tests are running against the database? Should we have a test database for this?

Comment (kmp3325) [inline src/test/java/accounting/data/TransactionDAOTest.java]: We could create an accounting-test database and just clear it every time...idk, I've been trying to find more elegant solutions, but there's too much involved in all of them.

**Comment by kmp3325** [comment]
We should merge this in soon because this gives us the investment endpoint.  I also changed the user to be the accounting and accounting_test user and updated the DAO test to user the test database.

**Comment by tuttleb** [comment]
This looks okay, I wish we had a better way to manage the test database. I think more coverage is worth the tradeoff

**Pull request merged**